scripts/zkai_attention_kv_ezkl_same_surface_export_probe.py

```
@@ -0,0 +1,484 @@
+        "git_commit",
+        "decision",
+        "source_shape",
+        "equality_conditions",
+        "equality_conditions_commitment",
+        "candidate_adapters",
+        "candidate_matrix_commitment",
+        "prior_evidence",
+        "non_claims",
+    }
+    if set(payload) != expected_fields:
+        raise SameSurfaceExportProbeError("payload field set mismatch")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)